packages/wallets/src/components/AccountsList/__tests__/AccountsList.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { PropsWithChildren } from 'react';
+import { APIProvider } from '@deriv/api';
+import { render, screen } from '@testing-library/react';
+import useDevice from '../../../hooks/useDevice';
+import { ModalProvider } from '../../ModalProvider';
+import AccountsList from '../AccountsList';
+
+jest.mock('../../../hooks/useDevice');
+const mockUseDevice = useDevice as jest.MockedFunction<typeof useDevice>;
+
+const wrapper = ({ children }: PropsWithChildren) => (
+    <APIProvider>
+        <ModalProvider>{children}</ModalProvider>
+    </APIProvider>
+);
+
+describe('AccountsList', () => {
```

Comment:
Probably deserves a little bit more test cases / expectations, 

I've removed majority of component code and this test is still passing. I shouldn't be able to remove majority of code from component without breaking the tests. 

![Screenshot 2024-01-29 at 14 04 19](https://github.com/binary-com/deriv-app/assets/141034155/82e3818d-df12-4e6a-8d9f-011fba7f5f6e)

Reply:
especially when removing props completely tests shouldn't still pass

Reply:
I cant really test the props @wojciech-deriv , because the current component not really using it but the onboarding component use it. More like integration test

Reply:
hmm, typically the prop should cause some effect, right? it is passed to trigger something? so can we check whatever is being triggered?

alternative option is to mock WalletMobileTourGuide component, and just display its flags, 

basically - props are component interface so we should be able to test them @farhan-nurzi-deriv

Reply:
thanks, yes, thats absolutely correct way to test it - ultimately the responsibility of this component is to pass props to and render child components. Whatever is happening inside child components, is not in scope of responsibility for this component/test, so do not need to be tested.